content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -34,7 +28,24 @@ For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overvi
 
 For information on installation and operations, see the *Digital Twin Manager -  Installation & operations guide*.
 
-### Device Management
+## Device Management
```

Comment:
Should be Heading level 3 ### and the following headings should have level 4 ####. Might conflict with Elias PR #56

Reply:
Ok. We deal with it.